portfolio/src/main/webapp/style.css

```diff
@@ -17,6 +21,47 @@ p {
 }
 
 img {
-  max-width: 450px;
+  max-width: 80%;
   max-height: auto;
 }
+
+#post-container {
+  border-style: solid;
+  border: 2px solid #000;
+  border-radius: 4px;
+  margin-top:20px;
+  margin-bottom: 20px;
+  overflow: hidden;
+  background-color: #e0dede;
+}
+
+#post-header {
+  margin-bottom: 30px;
+  overflow: auto; 
+  background-color: #aacfcf;
+}
+
+#post-content {
+  text-align: center;
+
```

Comment:
Remove extra new lines where possible to make it easier to read :)

Reply:
Done!